packages/adapters-library/src/adapters/protocols.ts

```diff
@@ -33,6 +33,7 @@ export const Protocol = {
   SparkV1: 'spark-v1',
   StakeWise: 'stakewise',
   Stargate: 'stargate',
+  StargateV2: 'stargate-v2',
```

Comment:
Do we want `stargate-v2` or do we want to group the new products inside `stargate`?

If we want to align our protocol ids with other providers, I don't think they have a stargate-v2.

Reply:
I dont know what debank have....lets wait for the ticket

Reply:
From their UI, it looks like everything is Stargate. Whether the API is the same or not, we don't know yet. 

I've taken the safe approach and all of them are under Stargate now.